packages/core/src/modules/message-pìckup/protocol/v2/V2MessagePickupProtocol.ts

```diff
@@ -82,21 +90,73 @@ export class V2MessagePickupProtocol extends BaseMessagePickupProtocol {
     return { message }
   }
 
+  public async deliverMessages(
```

Comment:
This should be called `createDeliveryMessage` or something I think. It does not deliver them